plugin.video.ted.talks/changelog.txt

```diff
@@ -0,0 +1,141 @@
+Version 4.3.1
```

Comment:
you may want to add a news section in addon.xml as changelog.txt isn't read in Kodi Krypton and newer.